internal/tofu/eval_import.go

```diff
@@ -49,9 +49,7 @@ func evaluateImportIdExpression(expr hcl.Expression, ctx EvalContext, keyData in
 			Summary:  "Invalid import id argument",
 			Detail:   `The import block "id" argument depends on resource attributes that cannot be determined until apply, so OpenTofu cannot plan to import this resource.`, // FIXME and what should I do about that?
 			Subject:  expr.Range().Ptr(),
-			//	Expression:
```

Comment:
Let's keep this comment.

Reply:
Let's extend this comment then, if it is actually meaningful. What should it reflect?

Reply:
I don't like unrelated changes in PRs, so I'd prefer to handle this separately.

When it is fixed, it should probably say something like that the diagnostic could be more useful and display better for the user if the expression and context (paired) are included.  This is a common issue anywhere we are setting Extra to DiagnosticCausedByUnknown.  I'm guessing this was copy-pasted from somewhere else in the past.